src/pages/CustomerRequisitionPage.js

```diff
@@ -156,8 +156,8 @@ export class CustomerRequisitionPage extends React.Component {
       ],
       [
         {
-          title: `${pageInfoStrings.months_stock_required}:`,
-          info: requisition.monthsToSupply,
+          title: `${pageInfoStrings.customer}:`,
+          info: requisition.otherStoreName ? requisition.otherStoreName.name : '',
```

Comment:
I'm guessing this is to give more room for the Customer (and below, Supplier) name. Good call probably, though there was some niceness to having the two 'editable' fields in the same column.